internal/service/organization/resource_organization_test.go

```diff
@@ -143,6 +143,24 @@ func TestAccConfigRSOrganization_Settings(t *testing.T) {
 	})
 }
 
+func TestAccOrganizationCreate_Errors(t *testing.T) {
+	var (
+		roleName    = "ORG_OWNER"
+		unknownUser = "65def6160f722a1507105aaa"
+	)
+	acc.SkipTestForCI(t) // affects the org
```

Comment:
as this test produces an error, does it really change anything in the org?

Reply:
You are right. I updated the comment